### PR TITLE
<doc>: update description of dump option.

### PR DESCRIPTION
**Pull request by fwhdzh**
Hello, I find that the description of dump option of tla2tools.jar is not consistent with https://lamport.azurewebsites.net/tla/current-tools.pdf. Therefore, I update it following the description in the pdf.

### CODE REVIEW

**Comment by lemmy** [comment]
@fwhdzh You will have to amend and sign the git commit, and then force-push it to this branch:

```shell
git commit --amend --sign
git push origin update-description-of-dump-option -f
```

The DCO checks needs to find something like `Signed-off-by: Markus Alexander Kuppe <github.com@...>` in the commit message.

**Comment by fwhdzh** [comment]
> @fwhdzh You will have to amend and sign the git commit, and then force-push it to this branch:
> 
> ```shell
> git commit --amend --sign
> git push origin update-description-of-dump-option -f
> ```
> 
> The DCO checks needs to find something like `Signed-off-by: Markus Alexander Kuppe <github.com@...>` in the commit message.

Thank you! I think I have finished it now.

File: tlatools/org.lamport.tlatools/src/tlc2/TLC.java [hunk 1]
```diff
@@ -1564,13 +1564,17 @@ private void printUsage()
...
 														"print various debugging information - not for production use\n",
 														true));
-    	sharedArguments.add(new UsageGenerator.Argument("-dump", "file",
-    													"dump all states into the specified file; this parameter takes\n"
-    														+ "optional parameters for dot graph generation. Specifying\n"
-    														+ "'dot' allows further options, comma delimited, of zero\n"
-    														+ "or more of 'actionlabels', 'colorize', 'snapshot' to be\n"
-    														+ "specified before the '.dot'-suffixed filename", true,
-    													"dot actionlabels,colorize,snapshot"));
+        sharedArguments.add(new UsageGenerator.Argument("-dump", "format file",
```
Comment (lemmy) [inline tlatools/org.lamport.tlatools/src/tlc2/TLC.java:1567]: Let's keep the the first part from the old version ("dump all states into the specified file."), followed by the new part from `current-tools.pdf`.

Comment (fwhdzh) [reply tlatools/org.lamport.tlatools/src/tlc2/TLC.java:1567]: I have updated my commit.

**Comment by lemmy** [comment]
Thanks!

**Pull request merged**